Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -2392,7 +2392,195 @@ kable(Well_Capacity)
+
+The wildfire hazard potential (WHP) map is a raster geospatial product produced by the USDA Forest Service, Fire Modeling Institute that can help to inform evaluations of wildfire hazard or prioritization of fuels management needs across very large landscapes. Our specific objective with the WHP map is to depict the relative potential for high-intensity wildfire that may be difficult to manage. To create the 2023 version, we built upon spatial datasets of wildfire likelihood and intensity generated with the Large Fire Simulator (FSim), as well as spatial fuels and vegetation data from LANDFIRE 2020 and point locations of past fire occurrence (ca. 1992 - 2020). With these datasets as inputs, we produced an index of WHP for the United States at a 270-meter resolution. We present the WHP map in two forms: 1) continuous integer values, and 2) five WHP classes of very low, low, moderate, high, and very high. Areas mapped with higher WHP values represent fuels with a higher probability of experiencing torching, crowning, and other forms of extreme fire behavior under conducive weather conditions, based primarily on landscape conditions at the end of 2020.
+
+This 2023 version of WHP was created from updated national wildfire hazard datasets of annual burn probability and fire intensity generated by the USDA Forest Service, Rocky Mountain Research Station with the large fire simulation system (FSim). Vegetation and wildland fuels data from LANDFIRE 2020 (version 2.2.0) were the primary inputs to the updated FSim modeling work and therefore form the foundation for this version of the WHP. As such, the data presented here reflect landscape conditions as of the end of 2020. LANDFIRE 2020 vegetation and fuels data were also used directly in the WHP mapping process, along with updated point locations of fire occurrence ca. 1992-2020. With these datasets as inputs, we produced an index of WHP for all of the conterminous United States at 270-meter resolution. We present the final WHP map in two forms: 1) continuous integer values, and 2) five WHP classes of very low, low, moderate, high, and very high. On its own, WHP is not an explicit map of wildfire threat or risk, but when paired with spatial data depicting highly valued resources and assets such as structures or powerlines, it can approximate relative wildfire risk to those specific resources and assets. WHP is also not a forecast or wildfire outlook for any particular season, as it does not include any information on current or forecasted weather or fuel moisture conditions. It is instead intended for long-term strategic fuels management.
+
+```{r}
+# GRAB MEAN WATER SUPPLY RANKING OF WHP IN THE WATER SUPPLY PYROME:
+  
+# continuous fire hazard potential, cropped to water supply pyrome:
+whp_cnt <- terra::rast("data/all/usfs_fire/whp2023_GeoTIF/whp2023_cnt_conus.tif") %>%
+  terra::crop(., sf::st_transform(water_supply_pyromes, 5070), 
+              mask = TRUE, touches = TRUE) %>%
+  na.omit()
```

Comment:
I don't think this `na.omit()` part is doing anything (at least on my end).

Reply:
that makes sense - I think that must be a byproduct of when I was doing this for all of IMR.